sdk/digitaltwins/Azure.DigitalTwins.Core/tests/DigitalTwinTests.cs

```diff
@@ -49,7 +48,14 @@ public async Task DigitalTwins_Lifecycle()
 
                 // update twin
                 string updateTwin = TestAssetsHelper.GetRoomTwinUpdatePayload();
-                await client.UpdateDigitalTwinAsync(roomTwinId, updateTwin).ConfigureAwait(false);
+
+                // create request options to force the update using "*" ifMatch value
+                var reqeustOptions = new RequestOptions
+                {
+                    IfMatch = new ETag("*")
+                };
+
+                await client.UpdateDigitalTwinAsync(roomTwinId, updateTwin, reqeustOptions).ConfigureAwait(false);
```

Comment:
how are we verifying?

Reply:
The request would be rejected and throw, if it failed.

Reply:
Note we're not trying to test the service. We're just making sure we pass along a payload and it gets sent/does not explode. We could maybe do more testing of specific scenarios (e.g. create, get etag, update, delete with original etag, ensure it fails) but that ventures into testing the service.